server/src/cmdline_utils.ts

```diff
@@ -37,6 +42,7 @@ interface CommandLineOptions {
 export function parseCommandLine(argv: string[]): CommandLineOptions {
   return {
     help: hasArgument(argv, '--help'),
+    ivy: hasArgument(argv, '--ivy'),
```

Comment:
nit: I would prefer this be named `--unstable-ivy` or similar, I am worried that someone will run this, expect it to be stable, and file issues.

Reply:
renamed to `--experimental-ivy`